ni/test/test_github.py

```diff
@@ -69,7 +69,8 @@ def test_bad_content_type(self):
             self.run_awaitable(github.Host.process(util.FakeServerHost(), request))
         self.assertEqual(cm.exception.response.status, 415)
 
-    def test_ping(self):
+    @mock.patch('ni.github.Host._verify_signature')
```

Comment:
I had a question about the patching....do you prefer explicitly setting the return_value of the mock _verify_signature object to be True or is a truthy value sufficient(by truthy, I mean the mocked objected is currently a Mock (or MagicMock instance hence it will pass a True)?